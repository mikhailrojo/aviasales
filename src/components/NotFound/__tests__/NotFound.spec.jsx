import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {NotFound} from '../index';

describe('Snapshot', () => {
	it('render NotFound', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(<NotFound />);
		expect(component).toMatchSnapshot();
	});
});
