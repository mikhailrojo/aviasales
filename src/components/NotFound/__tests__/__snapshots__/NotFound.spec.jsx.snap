// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render NotFound 1`] = `
<div>
  <styled.div />
  <p>
    К сожалению ничего не было найдено
  </p>
</div>
`;
